tools/awps-tunnel/client/src/panels/sections/SimpleClientSection.tsx

```diff
@@ -146,17 +128,16 @@ export const SimpleClientSection = ({ onStatusChange, url }: ClientPannelProps)
           </MessageBarBody>
         </MessageBar>
       )}
-      {connected && (
-        <p className="text-success">
-          <i>Connected</i>
-        </p>
-      )}
-      {!connected && (
-        <p className="text-info">
-          <i>Disconnected</i>
-        </p>
+
+      <div className="my-2">
```

Comment:
use a better class name?

Reply:
:D this is the bootstrap class name meaning margin 2 for y axis, `{property}{sides}-{size}` https://getbootstrap.com/docs/4.0/utilities/spacing/